src/core/atomic.d

```diff
@@ -27,6 +27,13 @@ enum MemoryOrder
      * Corresponds to $(LINK2 https://llvm.org/docs/Atomics.html#monotonic, LLVM AtomicOrdering.Monotonic)
      * and C++11/C11 `memory_order_relaxed`.
      */
+    rlx = 0,
```

Comment:
Using `relaxed` instead of this abbreviation would be more readable IMO

Reply:
I don't object to `relaxed`. But right now we use `acq` and `rel` instead of `acquire` and `release`. So it'd be a bit odd if `relaxed` is much longer all of a sudden.
You could argue that it should require more typing since it is easier to misuse and get synchronization that is too weak. But the answer to that would be don't use explicit access unless you know what you do. `SC` should be default and easy to use.